src/flamenco/runtime/tests/fd_exec_sol_compat.c

```diff
@@ -64,7 +63,7 @@ sol_compat_wksp_init( void ) {
   wksp = fd_wksp_new_anonymous( FD_SHMEM_NORMAL_PAGE_SZ, 65536UL * 8UL, fd_shmem_cpu_idx( fd_shmem_numa_idx( cpu_idx ) ), "wksp", 0UL );
   assert( wksp );
 
-  spad_mem = fd_wksp_alloc_laddr( wksp, FD_SPAD_ALIGN, spad_mem_max, 3 ); /* 1.28 GB */
+  spad_mem = fd_wksp_alloc_laddr( wksp, FD_SPAD_ALIGN, fd_spad_footprint( 128UL * fd_ulong_align_up( FD_ACC_TOT_SZ_MAX, FD_ACCOUNT_META_ALIGN ) ), 3 ); /* 1342191744 B */
```

Comment:
When this is merged, could you notify @nlgripto that our fuzzing memory requirements have slightly increased? 🙏